samples/nrf9160/aws_fota/src/main.c

```diff
@@ -0,0 +1,571 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr.h>
+#include <stdio.h>
+#include <uart.h>
+#include <string.h>
+
+#include <net/mqtt.h>
+#include <net/socket.h>
+#include <net/aws_fota.h>
+#include <lte_lc.h>
+#if defined(CONFIG_BSD_LIBRARY)
+
+#include "nrf_inbuilt_key.h"
+#endif
+
+#define CONFIG_NRF_CLOUD_SEC_TAG 16842753
+#define CONFIG_AWS_HOSTNAME "a25jld2wxwm7qs-ats.iot.eu-central-1.amazonaws.com"
+
+#include "aws_cert/rootca_ec.h"
+#include "aws_cert/private_key.h"
+#include "aws_cert/pubcert.h"
+
+#if !defined(CONFIG_CLIENT_ID)
+#define IMEI_LEN 15
```

Comment:
This feels like it should be present in a header file somewhere.

Reply:
Unfortunately it does not.

Reply:
It unfortunately does not